scipy/stats/tests/test_distributions.py

```diff
@@ -3026,21 +3028,43 @@ def test_isf(self):
 
 
 class TestLevyStable:
+    @pytest.mark.slow
+    def test_rvs(self):
+        for param in ["S0", "S1"]:
+            # Various arbitrary pairs. It's
+            # important to have variations of
+            # alpha, beta == 1 and !=1.
+            for alpha, beta in [
+                (1.0, 0),
+                (1.0, -0.5),
+                (1.5, 0),
+                (1.9, 0.5),
+            ]:
+                # check standardized and non-standardized
+                for gamma, delta in [(1, 0), (3, 2)]:
+                    stats.levy_stable.parameterization = param
+                    ls = stats.levy_stable(
+                        alpha=alpha, beta=beta,
+                        scale=gamma, loc=delta)
+                    _, p = stats.kstest(
+                        ls.rvs(size=1500, random_state=1234), ls.cdf)
+                    assert_equal(p > 0.05, True)
```

Comment:
If you have a single test run over many parameter values, you should use [@pytest.mark.parameterize](https://docs.pytest.org/en/6.2.x/parametrize.html). This makes it easy to run the test over all of the parameter values even if some fail and then see exactly which cases failed.